docs/dApps/unity/quickstart.md

```diff
@@ -0,0 +1,373 @@
+---
+title: Quickstart
+authors: Tim McMackin
+last_update:
+  date: 4 December 2023
+---
+
+Follow these steps to install the Tezos SDK for Unity in an existing Unity project and start using it.
+
+These instructions cover:
+
+- Installing the SDK into an existing Unity project
+- Testing that the SDK works in your project
+- Connecting to a user's Tezos wallet
+- Creating and managing tokens
+- Prompting the user to sign messages
+
+## Installing the SDK
+
+1. Make sure that you have Unity Editor version 2021.3.23f1 or later.
+
+1. Install a Tezos-compatible wallet on a mobile device that has a camera and can scan QR codes.
+For instructions, see [Installing and funding a wallet](../../developing/wallet-setup).
+
+1. Switch the wallet to Ghostnet and use the [Ghostnet faucet](https://faucet.ghostnet.teztnets.xyz/) to get some tez that you can use to send transactions to your Unity projects.
+
+1. Install the SDK from GitHub:
```

Comment:
We should simply tell users to follow the installation steps on GitHub. That way we only have 1 place to maintain as things change.

Reply:
Left the installation section in but now it's a link to github and a mention of setting up a wallet.